core/arch/arm32/sta/sst_km_tests.c

```diff
@@ -0,0 +1,353 @@
+/*
```

Comment:
I think I'd like to see this put in optee_test.git instead. Either in the crypto ta or the storage ta.